ddtrace/contrib/openai/patch.py

```diff
@@ -0,0 +1,459 @@
+import os
+import random
+import sys
+
+from ddtrace import config
+from ddtrace.internal.constants import COMPONENT
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils.formats import asbool
+
+from . import _log as ddlogs
+from .. import trace_utils
+from .. import trace_utils_async
+from ...pin import Pin
+from ..trace_utils import wrap
+from ._metrics import stats_client
+
+
+config._add(
+    "openai",
+    {
+        "logs_enabled": asbool(os.getenv("DD_OPENAI_LOGS_ENABLED", False)),
```

Comment:
TODO: test logs with vcrpy, requires the fix to flush the logs within the test case